modules/sdk-coin-avaxp/src/lib/iface.ts

```diff
@@ -64,7 +67,10 @@ export const SECP256K1_Transfer_Output = 7;
 
 export const ADDRESS_SEPARATOR = '~';
 export const INPUT_SEPARATOR = ':';
-export type DeprecatedTx = PMVTx | EMVTx;
-export type Tx = avaxSdkPvmSerial.AddPermissionlessValidatorTx;
-export type BaseTx = PMVBaseTx | EVMBaseTx;
-export type Output = TransferableOutput | EVMOutput;
+export type DeprecatedTx = PVMTx | EVMTx;
+export type DeprecatedBaseTx = DeprecatedPMVBaseTx | DeprecatedEVMBaseTx;
+export type Tx = DeprecatedTx | pvmSerial.BaseTx | UnsignedTx;
```

Comment:
we should remove pvmSerial.BaseTx to avoid confusion, I don't see any use-case when we would use pvmSerial.BaseTx directly, it will always be wrapped inside UnsignedTx